### PR TITLE
docker included v11.0

### PR DESCRIPTION
**Pull request by CodeWebWeaver**

### CODE REVIEW

File: src/test/java/com/parkhomovsky/bookstore/BookstoreApplicationTests.java [hunk 1]
```diff
@@ -1,13 +1,8 @@
...
-import org.junit.jupiter.api.Test;
 import org.springframework.boot.test.context.SpringBootTest;
 
 @SpringBootTest
 class BookstoreApplicationTests {
 
-    @Test
-    void contextLoads() {
-    }
-
```
Comment (nacenik) [inline src/test/java/com/parkhomovsky/bookstore/BookstoreApplicationTests.java:12]: Why did you delete this? 
Do not touch this test.

File: src/main/resources/db/changelog/changes/07-insert-admin-in-users-table.yaml [hunk 2]
```diff
@@ -1,29 +1,29 @@
...
+                  value: PonyDestroyer
+              - column:
+                  name: password
+                  value: I$J(QJ#T(QIG#Q*G(Q#JIRARJAK@(JRHI==wf
+        - insert:
+            tableName: users_roles
+            columns:
+              - column:
+                  name: user_id
+                  valueComputed: 1
```
Comment (nacenik) [inline src/main/resources/db/changelog/changes/07-insert-admin-in-users-table.yaml:26]: For future: this is bad idea to change sth in migration scripts that were pushed. If you have a db on prod, this change will brake your service at all.

File: pom.xml [hunk 3]
```diff
@@ -70,14 +70,12 @@
...
-            <groupId>com.h2database</groupId>
-            <artifactId>h2</artifactId>
-            <scope>test</scope>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-docker-compose</artifactId>
         </dependency>
         <dependency>
-            <groupId>com.mysql</groupId>
-            <artifactId>mysql-connector-j</artifactId>
-            <version>8.1.0</version>
```
Comment (nacenik) [inline pom.xml:80]: Don't you use mySQL on your local anymore?

File: docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+      POSTGRES_PASSWORD: $POSTGRES_PASSWORD
+      POSTGRES_DB: $POSTGRES_DATABASE
+    ports:
+      - "${POSTGRES_LOCAL_PORT}:${POSTGRES_DOCKER_PORT}"
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready -U postgres"]
+      interval: 30s
+      timeout: 30s
+      retries: 3
+  app:
```
Comment (nacenik) [inline docker-compose.yml:21]: add empty line between services. This improve code readability

File: src/main/resources/application.properties [hunk 5]
```diff
@@ -1,8 +1,3 @@
-spring.datasource.url=jdbc:mysql://localhost:3306/bookstore?serverTimezone=UTC
-spring.datasource.username=root
-spring.datasource.password=vjzctvmz33
-spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
```
Comment (nacenik) [inline src/main/resources/application.properties:4]: what about local db?

**Review by nacenik — CHANGES_REQUESTED** [review]
good. Let's improve a little bit

File: docker-compose.yml [hunk 6]
```diff
@@ -0,0 +1,40 @@
+version: "3.0"
+
+services:
+  postgresdb:
+    image: postgres
+    container_name: bookstore_container
```
Comment (boroda4436) [inline docker-compose.yml:6]: Do you need it? If yes, please edit the name to identify this is a DB

**Review by boroda4436 — APPROVED** [review]
